examples/node-oauth/README.md

```diff
@@ -1,73 +1,108 @@
+- [Axios](https://axios-http.com/docs/intro)
 
-ℹ️ Note:
+# ✅ Prerequisites <a name="prerequisites"></a>
 
-- Comments with `-->`or `<--` indicate a significant step in the flow. \
-  Comments without these markers are for additional reference about the code.
+- Create a [new app in Miro](https://miro.com/app/settings/user-profile/apps)
+- You have a [Miro account](https://miro.com/signup/).
+- You're [signed in to Miro](https://miro.com/login/).
+- Your Miro account has a [Developer team](https://developers.miro.com/docs/create-a-developer-team).
+- Your development environment includes [Node.js 14.13](https://nodejs.org/en/download) or a later version.
+- All examples use `npm` as a package manager and `npx` as a package runner.
+- Install [ngrok](https://ngrok.com/).
 
-For the full guide on Miro's OAuth 2.0 flow, [see the documentation](https://developers.miro.com/docs/getting-started-with-oauth).
+# 📖 Associated Developer Tutorial <a name="tutorial"></a>
 
-This app uses Express for a local server, Localtunnel for exposing localhost, Node.js, and Axios for making HTTP requests.
+> To view a more in depth developer tutorial
+> of this topic see the [OAuth 2.0 Miro tutorial](https://developers.miro.com/docs/getting-started-with-oauth) on Miro's Developer documentation.
 
-### Prerequisites
+# 🏃🏽‍♂️ Run the app locally <a name="run"></a>
 
-- [Create a new app in Miro](https://miro.com/app/settings/user-profile/apps).
-- Install [Localtunnel](https://www.npmjs.com/package/localtunnel) or a similar tool, such as [ngrok](https://ngrok.com/).
+1. Run `npm install` to install dependencies.
 
-### How to start
+2. Run `ngrok http 3000` and copy the forwarding address. It should look something like:
 
-1. Clone or download repo.
-2. `cd` to the repo root folder.
-3. `npm install` to install dependencies.
-4. Create an `.env` file in the root folder, and set the following variables:
+`https://1ce8-45-94-204-74.ngrok-free.app`. This will be the value for `redirectURL` in your `.env` file.
+
+3. Rename the `.sample.env` file to `.env` and then fill in the values.
+
+> To find your client ID and secret, go to `Your apps`, and then select the app you just created to access its settings page. On the app settings page, go to `App Credentials`, and copy the app Client ID and Client secret values.
+> To find your `boardId` Go to your desired Miro board, copy the board ID from the URL, and paste it to your .env file boardId variable. For more info on how to get your board ID, see the [community question](https://community.miro.com/developer-platform-and-apis-57/where-can-i-find-board-id-3154).
+> The `redirectURL` is just the forwarding address from step 2. Once your `.env` file is completed, it should look similar to this:
 
 ```
-clientID=<YOUR_CLIENT_ID>
-clientSecret=<YOUR_CLIENT_SECRET>
-redirectURL=<YOUR_REDIRECT_URL>
-boardId=<MIRO_BOARD_ID>
+clientID=12345678910
+clientSecret=12345678910abcdefg
+redirectURL=https://01-11-11-2011-74.ngrok-free.app
```

Comment:
Should this be localhost?

Reply:
@mettin from when I talked to Will, it seemed like this sample was positioned as one where you need a hosted redirect. Is that correct @bishopwm ?

Reply:
When I originally created this app, I used ngrok, but maybe that was before localhost was supported.

IMO it's good to have at least one example that uses something other than localhost, but don't have a super strong opinion :)

Reply:
I've just tested this and it works fine on both localhost and with ngrok. 

I think generally it's best to have the most frictionless path up front (localhost) and then we can also have a separate guide for ngrok since it takes a bit more work to setup. 

I can work on this and just add in one more commit later today.

I wonder if it's worth having a separate name for a sample app which is using a "hosted" solution so that it can easily be identified from the other 20+ sample apps we have. Otherwise I'm not quite sure how a developer would know that the node-oauth sample is the one which has a guide for ngrok / non-localhost redirect URL. 

Any thoughts here @mettin @bishopwm ?

Reply:
That sounds good to me @horeaporutiu . I agree localhost is easier. 

We may not even need to have an ngrok example, but could add an item to our backlog to create a guide around it in the future if we think it's valuable

Reply:
Since I already have the demo videos and instructions for ngrok, I will just add it as an optional step in the node-oauth guide if that works for you both?

And then later on we can decide if we want to have a separate sample for this or if we just want to leave it as is. If a developer asks about using a hosted URL we can always point to this optional part of the guide. Since it is probably the most straight forward sample w/ least amount of code we have I think this is a good candidate to show both localhost and ngrok.

Reply:
Sounds good to me 👍

Reply:
Ok - it's been updated now. Lmk if it looks ok with you both @mettin @bishopwm